documents/motions/10.2024.4 - Appeals Committee.md

```diff
@@ -0,0 +1,33 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 10.2024.4  
+**Subject:** WCA Appeals Committee  
+**Intent:** Rights and duties of the WCA Appeals Committee  
+**Submitted by:** WCA Board  
+**Date:** TBC, 2024  
+
+# Motion
+
+The WCA Appeals Committee is an Advisory Committee of the WCA.
+
+1. The WCA Appeals Committee has the following rights and duties:
+   1. Forming the Board of Appeal to carry out the appeals processes described in [WCA Motion : Disputes](wca{documents/motions/16.2021.1 - Disputes.pdf}).
+      1. The Board of Appeal shall be made up of three members of the WCA Appeals Committee.
+      2. The Leader of the WCA Appeals Committee determines the members of the Board of Appeal for each dispute.
+      3. Members of the WCA Appeals Committee who have a conflict of interest for a specific dispute shall be ineligible from selection to that specific Board of Appeal.
+      4. The Board of appeal shall have all the rights and duties assigned to the WCA Appeals Committee and shall have the full right to represent the committee in the appeals process.
+   2. Performing independent investigations regarding appealed incidents.
+   3. Making decisions on disciplinary actions in relation to appealed incidents.
+   4. Informing the WCA Board on appeals, investigations, decisions, and disciplinary actions.
+   5. Publishing public information on the appeals, investigations, decisions, and disciplinary actions where necessary.
+   6. Managing an administration of the appeals, investigations, the reports, the decisions, and the disciplinary actions.
```

Comment:
"Managing an administration" seems odd, Maybe just "Administering" or "Maintaining records".  but tbh not sure of the intent of this sentence.  Is it necessary to codify that they will keep records of their work?

Reply:
My view is it is 1) Administering of the outcomes of appeals (e.g. tracking that a ban is put in force), 2) maintaining a documents record of appeals etc. I have proposed updated wording.